text_formatter/check.py

```diff
@@ -0,0 +1,35 @@
+"""
+Check the text types, to see if they can be used on other
+``text_formatter`` functions.
+"""
+
+# import the required std libraries
+import string
+from typing import Optional
+
+# get a custom exception from ``text_formatter.exceptions``
+from text_formatter.exceptions import InvalidString
```

Comment:
Since this line is taking `text_formatter.exceptions.InvalidString`, we must merge #25 before we merge this one.

Reply:
#25 has been succesfully merged, now we can merge this other pull request.